packages/lumx-react/src/components/button/ButtonRoot.tsx

```diff
@@ -161,14 +169,20 @@ const ButtonRoot: React.FC<ButtonRootProps> = (props) => {
 
     if (!isEmpty(props.href)) {
         return (
-            <a {...forwardedProps} ref={buttonRef as RefObject<HTMLAnchorElement>} className={buttonClassName}>
+            <a
+                {...forwardedProps}
+                ref={buttonRef as RefObject<HTMLAnchorElement>}
+                className={buttonClassName}
+                aria-disabled={isDisabled}
```

Comment:
I'm not sure I see the use case where we would want a disabled link?
If anything, if badly used wouldn't it confuse the user more than it would help him?

For example
```javascript
<Button href="http://google.com"  isDisabled>My profile</Button>
```
focusing this button will cause the screen reader to read
`My profile. Link Disabled` indicating to the user that the link isn't usable. 
But nothing prevents him to press the Enter key and still follow the link.

Maybe there is a use case that I can't think of though ? ^^'

Reply:
Ok we could either remove the href when `isDisabled` is set or just render a disabled button (but the DOM would alternate between <a> and <button> when toggling the `isDisabled` prop)
What do you think?

Reply:
Yeah you're right, that also might be tad bit confusing. x_x 

For screen reader purpose I would probably, as you said, keep the "a" tag and remove the href attribute.
This way, the screen reader whould either read `My profile. Link` or `My profile. Link disabled`, which correctly describes the link's state. Eslint won't be very fond of it...

I don't now if we want to inforce it through the DS, or leave the responsability to the user to do so though ?
Also, we probably should apply some styling to show that the button is supposed to be disabled ?